src/conduit/util.py

```diff
@@ -0,0 +1,36 @@
+"""Utilities for conduit app."""
```

Comment:
`util` is in most cases a bad module name. These kinds of files always end up a sink of random code that should really be elsewhere.

Reply:
I don't see where `expandvars_dict` is being used?

Reply:
Right. I forgot to `git stash apply`. `expandvars_dict` is being used in `src/conduit/migrations/env.py` and `src/conduit/__init__.py`. I could not find a better place to define it. An option was to define it as `conduit.expandvars_dict` but this way, there would be chances of recursive imports.

Reply:
Another option was to update `pyramid_heroku` itself which I tried but found it would break existing logic. As shell variables don't work with multi level variable definition.

```bash
FOO=foo
BAR='${FOO}bar'

echo ${BAR:4:3}
O}b
# Instead of "ar"
```

Reply:
Where do we use multi-level definitions?

Reply:
I assumed we (or `pyramid_heroku` users) are probably using it by reading the comments.
`Env vars are expanded twice, so you can have (1-level) nesting. Example:`

https://github.com/niteoweb/pyramid_heroku/blob/b0c20cebebc9bb5bb35c505b3df89c1352800aba/pyramid_heroku/__init__.py#L33-L40

Should I reopen https://github.com/niteoweb/pyramid_heroku/pull/26 ?

Reply:
Ah yes, I think we need those for Review Apps support in EBN. However, it is possible we no longer need them. 

In any case, if you can reopen and fix #26, that would be the safest approach.

Reply:
I think you meant https://github.com/niteoweb/pyramid_heroku/pull/26 😄 Alright.. I'll reopen that.

Reply:
Cat reopen that since I force pushed commits. Opened https://github.com/niteoweb/pyramid_heroku/pull/27 instead.